spec/expression.dd

```diff
@@ -514,8 +514,9 @@ $(GNAME ShiftExpression):
         $(D >)$(D >)$(D >) is an unsigned right shift.
     )
 
-    $(P It's illegal to shift by the same or more bits than the size of the
-        quantity being shifted:
+    $(P If the quantity is shifted by the same or more bits than the size of the
+	quantity, it is implementation defined whether a diagnostic is issued
```

Comment:
"it is implementation defined whether a diagnostic is issued"

->

"the behavior is implementation defined"

because this also covers getting a silent result (such as zero), or aborting execution.